indexer/borders.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+#include "geometry/region2d.hpp"
+
+#include <cstdint>
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace indexer
+{
+class Borders
```

Comment:
Please add a comment to this class: what it holds, where it is supposed to be used and what the inner structure is . The meaning of inner/outer is not immediately obvious.

Reply:
done